Doc/whatsnew/3.13.rst

```diff
@@ -334,6 +334,12 @@ Removed
   attribute instead.
   (Contributed by Nikita Sobolev in :gh:`105546`.)
 
+* Remove deprecated ``re.template`` function
+  and ``re.T`` and ``re.TEMPLATE`` flags.
+  Also removes ``SRE_FLAG_TEMPLATE`` constant
+  from ``sre_constants`` and ``re._constants``.
+  (Contributed by Nikita Sobolev in :gh:`105687`.)
```

Comment:
You might want to credit Serhiy who attempted first to remove it.

Reply:
Of course!

Reply:
I added Serhiy as the first name 👍